### ISSUE TITLE
Add docstrings to the file.

### ISSUE THREAD

**Issue by vibhor98** [vibhor98/Intranet-Chat-Box#7]
The  file  `chat_server.py`  needs  docstrings.

**Comment by hi15** [comment]
I will work on it.

**Comment by vibhor98** [comment]
Hi Hamer,

Sure you can work on it. This is open source and any one can send the PR.

**Comment by vesakivisto** [comment]
Hey,

There seems to be some docstrings already. Should all the methods have docstrings?

**Comment by vibhor98** [comment]
Yes, all the methods should contain the docstrings. Moreover, add what the
method takes as an argument, what it returns and what exceptions(if any)
 it handles.
